content/workshops/frontend.md

```diff
@@ -0,0 +1,52 @@
+{
+    "type": "tutorial",
+    "layout": "type",
+    "title": "Cdev Basics Workshop",
+    "linktitle": "fullstack", 
+    "card_icon": "ti-desktop",
+    "card_body": "Learn how to create APIs using Cdev",
+    "weight": "3"
+}
+## Who is this workshop for?
+{{<break 1>}}
+This workshop is for anyone who has some basic web-development knoledge and an interest in serverless development.
+{{<break 1>}}
+## Goals of the Workshop
+In this workshop we will teach you the basics of using the Cdev SDK and how to create and host a static site.

```

Comment:
remove 'and how to create' because this one will assume they have a site to already host (i.e their resume website or something equivalent)